code/api/src/seeders/20200825050120-quiz.js

```diff
@@ -0,0 +1,378 @@
+      {
+        name: "Rocker Flower Boots",
+        slug: "rocker-flower-boots",
+        description: "Stomp the roses",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.male.id,
+        image: "/images/quiz/rocker-shoes2.png",
+        style: "rocker",
+        isSurvey: true,
+        category: "shoes",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Rocker Top",
+        slug: "rocker-top",
+        description: "Look what you made me do",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.female.id,
+        image: "/images/quiz/rocker-top1.png",
+        style: "rocker",
+        isSurvey: true,
+        category: "top",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Rocker Combo",
+        slug: "rocker-combo",
+        description: "I see the shadows on my face",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.male.id,
+        image: "/images/quiz/rocker-top2.png",
+        style: "rocker",
+        isSurvey: true,
+        category: "top",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Artsy Result",
+        slug: "artsy-result",
+        description: "Artsy",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.female.id,
+        image: "/images/quiz/result-artsy.png",
+        style: "artsy",
+        isSurvey: true,
+        category: "result",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Bohemian Result",
+        slug: "bohemian-result",
+        description: "Bohemian",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.female.id,
+        image: "/images/quiz/result-bohemian.png",
+        style: "bohemian",
+        isSurvey: true,
+        category: "result",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Business Result",
+        slug: "business-result",
+        description: "Business",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.female.id,
+        image: "/images/quiz/result-business.png",
+        style: "business",
+        isSurvey: true,
+        category: "result",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: "Rocker Result",
+        slug: "rocker-result",
+        description: "Rocker",
+        type: params.product.types.cloth.id,
+        gender: params.user.gender.female.id,
+        image: "/images/quiz/result-rocker.png",
+        style: "rocker",
+        isSurvey: true,
+        category: "result",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+    ]);
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.bulkDelete("products", null, {});
+  },
+};
```

Comment:
If you want to edit an existing file, just run `node_modules/.bin/sequelize db:seed:undo:all` and this will undo every single seed. Then run `npx sequelize-cli db:seed:all`

Reply:
I've been manually removing seeds directly in psql. This is going to be super helpful going forward.